pkg/process/init.go

```diff
@@ -91,6 +92,7 @@ func NewRunc(root, path, namespace, runtime, criu string, systemd bool) *runc.Ru
 		Root:          filepath.Join(root, namespace),
 		Criu:          criu,
 		SystemdCgroup: systemd,
+		Debug:         logrus.GetLevel() == logrus.DebugLevel,
```

Comment:
suggest a todo for making this a separately configurable option... maybe also modify the docs to indicate debug level will flow through to the runc runtime